lib/server.js

```diff
@@ -164,6 +184,64 @@ module.exports = function createServer(config, log) {
   )
 
   api.post(
+    '/checkAuthenticated',
+    function (req, res, next) {
+      var action = req.body.action
+      var email = req.body.email
+      var ip = req.body.ip
+      var uid = req.body.uid
+
+      if(!action || !ip || !uid){
+        var err = {code: 'MissingParameters', message: 'action, email, ip and uid are all required'}
```

Comment:
from mtg: let's not worry about that for now.  The limits here are a lot more generous than on e.g. number of login attempts.  So If we have some QA environment that's hammering us hard enough to trigger one of these rate-limits, that's a good signal that we should fix the QA environment rather than allow it to hammer harder.